sources/packages/api/src/route-controllers/education-program/education-program.controller.ts

```diff
@@ -186,7 +205,29 @@ export class EducationProgramController {
       approvalStatus: educationProgram.approvalStatus,
       programIntensity: educationProgram.programIntensity,
       institutionProgramCode: educationProgram.institutionProgramCode,
+      institutionId: educationProgram.institution.id,
```

Comment:
Can you please elaborate on why do we need institutionId and institutionName returned on this method?